src/components/FormBuilder.tsx

```diff
@@ -280,7 +287,7 @@ export class FormBuilder extends React.Component<IFormBuilderProps, {}> {
 
         const component = React.createElement(fieldDef.builder, fieldBuilderProps);
         return (
-            <div className='form-builder-field' key={field.id}>
+            <div ref={this.attachFieldRef} className='form-builder-field' key={field.id}>
```

Comment:
This behavior could be strange. But it could be useful for end-users that add to the end of the form when building because it automatically scrolls to the bottom for you.